src/frequenz/sdk/config/_config_managing.py

```diff
@@ -130,6 +157,12 @@ def _read_config(self) -> abc.Mapping[str, Any]:
             except ValueError as err:
                 _logger.error("%s: Can't read config file, err: %s", self, err)
                 error_count += 1
+            except FileNotFoundError:
+                # It is ok for config file to don't exist.
+                _logger.info(
+                    "%s: Config file %s not found. Ignoring it.", self, config_path
```

Comment:
Just to make it more explicit what I'm suggesting to change in this PR:
```suggestion
            except OSError as err:
                # It is ok for config file to don't exist.
                _logger.error(
                    "%s: Error reading config file %s (%s). Ignoring it.", self, config_path, err,
```

Reply:
fixed